addon/components/paper-input.js

```diff
@@ -3,61 +3,74 @@ import BaseFocusable from './base-focusable';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
 import FlexMixin from 'ember-paper/mixins/flex-mixin';
 
+const { $, computed, isPresent, isArray, Logger, A } = Ember;
+
 export default BaseFocusable.extend(ColorMixin, FlexMixin, {
   tagName: 'md-input-container',
   classNames: ['md-default-theme'],
-  classNameBindings: ['hasValue:md-input-has-value', 'focus:md-input-focused', 'isInvalid:md-input-invalid', 'iconFloat:md-icon-float'],
+  classNameBindings: [
+    'hasValue:md-input-has-value',
+    'focus:md-input-focused',
```

Comment:
BaseFocusable handles this. Also, `focus` isn't used anymore. BaseFocusable uses `focused`.